tutorial/reverse-proxy.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Reverse Proxy
+parent: Malibu Tutorial
+nav_order: 31
+
+---
+
+In computer networks, a reverse proxy is the application that sits in front of back-end applications and forwards client requests to those applications.
+
+To implement reverse proxy we are using an npm package named `http-proxy`. Here is a sample implementation of reverse proxy:
```

Comment:
Dont we need to implement both the cases you worked on? One with ignorePath and one without?